<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckOrderCategoryWidget</class>
 <widget class="QWidget" name="CheckOrderCategoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;
color:#999999;</string>
        </property>
        <property name="text">
         <string>日期筛选：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startDateEdit">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:12px;
color:#999999;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #6c6c6c;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endDateEdit">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:12px;
color:#999999;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="SearchEdit" name="searchWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>262</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>262</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font-size:14px;
color:#999999;</string>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CheckOrderCategoryListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QListView{
border:none;
background-color:transparent;
}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPageButton">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image: url(:/setImage/right.png);
}

QPushButton:pressed{
border-image: url(:/setImage/right_press.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchEdit</class>
   <extends>QWidget</extends>
   <header>SearchEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckOrderCategoryListView</class>
   <extends>QListView</extends>
   <header>CheckOrderCategoryListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
